src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -483,6 +486,10 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                 case MessageID.IDS_FeatureSwitchOnBool: // Checked in the binder.
                     return LanguageVersion.CSharp2;
 
+                // PROTOTYPE(extended-property-patterns)
+                case MessageID.IDS_FeatureExtendedPropertyPatterns:
```

Comment:
nit: preview features go at the top (see `IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction`). Yes, it'll like be a conflict, albeit simple to address. Then no PROTOTYPE marker is necessary

Reply:
> Yes, it'll like be a conflict, albeit simple to address.

I've done this before and it conflicted with every single feature that merged in. I'd prefer we remove and address proto comments all at once as the last step.